pkg/controller/integration/integration_controller.go

```diff
@@ -78,6 +79,14 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		UpdateFunc: func(e event.UpdateEvent) bool {
 			oldIntegration := e.ObjectOld.(*v1alpha1.Integration)
 			newIntegration := e.ObjectNew.(*v1alpha1.Integration)
+
```

Comment:
My understanding is that this statement is already logged in the `Reconcile` method.

Reply:
oh yeah, that's a leftover from the rebase, will fix it

Reply:
done